src/modules/airspeed_selector/airspeed_selector_params.c

```diff
@@ -1,4 +1,12 @@
 
+/**
+ * Airspeed validator/selector enable
```

Comment:
Can we add here in the param description when to disable the module? And what the effects of it being disabled are?